src/views/About/index.jsx

```diff
@@ -21,7 +21,6 @@ const About = () => {
           <CardHeader
             className={styles.about_title}
             title="Conceived and Built at Gordon College"
-            subheader="By Students for Students"
```

Comment:
Just checking, was this deleted to make the header look right?  I just want to make sure this sub-header is still visible on the page, since it's an important part of the about page.